### PR TITLE
(RE-13380) Enable hardened runtime for Apple Notarization and update signing server.

### PR DESCRIPTION
**Pull request by jackie-kinsler**

### CODE REVIEW

**Comment by joshcooper** [comment]
jenkins please test this on osx11-64a,osx1015-64a

**Comment by jackie-kinsler** [comment]
jenkins please test this on osx11-64a,osx1015-64a

**Comment by jackie-kinsler** [comment]
jenkins please test this on osx11-64a,osx1015-64a

**Comment by jackie-kinsler** [comment]
jenkins please test this on osx11-64a,osx1015-64a

File: configs/platforms/osx-10.15-x86_64.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@
   plat.provision_with 'cd /etc/homebrew'
   plat.provision_with 'su test -c \'echo | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"\''
   plat.provision_with 'sudo chown -R test:admin /Users/test/'
+  plat.provision_with 'ssh-add -l'
```
Comment (e-gris) [inline configs/platforms/osx-10.15-x86_64.rb:19]: I'd add above: `plat.provision_with 'echo ============ SSH DEBUG STATEMENT FOLLOWS ======='`

**Comment by jackie-kinsler** [comment]
jenkins please test this on osx11-64a,osx1015-64a

**Comment by joshcooper** [comment]
I'm going to close this out because of the merge conflicts. We'll create a new PR when we're ready to start on the Notarization work.

**Pull request closed without merging**